content/dogepedia/faq/dogecoin-has-no-developers.md

```diff
@@ -5,9 +5,10 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-Oh yes it does, and it always had. Just, since Dogecoin main purpose has been to be a currency since its birth, and since it was forked from Litecoin and then reforked from Bitcoin, with changed parameters to make it more suitable as a cryptocurrency, it means it inherited already feature-complete code bases. So, for many years, instead of having new shiny features, the development team worked mostly as maintainers of what was already there. Not once, in its history, Dogecoin did not have a group of developers working on it.
-
-In the future, the plan is to focus more efforts and resources to enhance Dogecoin, and make it fullfill its aim to become a crypto of the people also in its technical implementation. This will require a combination of part time contributors, volunteers, but also full time developers and engineers. Alongside, the foundation and the advisors will provide a "roadmap" to aim at possible goals: roadmap that will never be authoritative or normative, but an indication of where to go.
+Oh, yes it does - and it always has!  
 
+Dogecoin’s primary purpose has always been to be used as a currency, and since it was forked from Litecoin and then reforked from Bitcoin — with changed parameters to make it more suitable as a cryptocurrency — Dogecoin inherited already feature-complete code bases. So, for many years, instead of having new shiny features, the development team worked mostly as maintainers of what was already there. Not once, in its history, Dogecoin did *not* have a group of developers working on it. 
```

Comment:
```suggestion
Dogecoin’s primary purpose has always been to be used as a currency. Dogecoin was originally a fork of a cryptocurrency called Luckycoin - itself a fork of Litecoin. Billy Markus, co-creator of Dogecoin with Jackson Palmer, acted as a solo developer for the first Dogecoin Core releases. In 2014, Billy and Jackson left Dogecoin development, and a new development team was formed.  Over the years, the Dogecoin development team has consisted of a small group of maintainers coordinating the work of more than 40 contributors.

In 2015, starting from Dogecoin Core 1.10.0, the Dogecoin development team decided to re-fork Dogecoin off Bitcoin, re-applying Litecoin changes on top of the Bitcoin codebase. Thus, throughout its history, Dogecoin inherited already feature-complete code bases, and it has shared a good part of its identity with Bitcoin. Its uniqueness came from changed parameters, optimizations, and parts of the code that were not inherited from Litecoin or Bitcoin. These changes were always aimed at making Dogecoin a viable currency for day to day spending, with a focus on utility and ease of use.
```

Reply:
The original paragraph inherited from my stub contained some imprecise statements. I fixed it and expanded it in my suggestion.